src/plugins/cargo-workspace.ts

```diff
@@ -82,17 +72,6 @@ interface CrateInfo {
  * into a single rust package.
  */
 export class CargoWorkspace extends WorkspacePlugin<CrateInfo> {
-  constructor(
-    github: GitHub,
-    targetBranch: string,
-    repositoryConfig: RepositoryConfig,
-    options: WorkspacePluginOptions = {}
-  ) {
-    super(github, targetBranch, repositoryConfig, {
```

Comment:
@fasterthanlime, I'm curious what your motivation was for this default behavior?

@chingor13 question on my mind is whether we should still make this configurable.

Reply:
Yes, we can expose the option in the manifest, but will do it separately.